CONTRIBUTING.md

```diff
@@ -1,164 +1,157 @@
 # Contributing
 
+This guide covers our development workflow, coding standards, and how to get your changes merged.
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+2. [How to Contribute](#how-to-contribute)
+3. [Code Quality Guidelines](#code-quality-guidelines)
+4. [Submitting Changes](#submitting-changes)
+5. [Localization (i18n)](#localization-i18n)
+6. [Copyright Attribution](#copyright-attribution)
+
+## Getting Started
+
+### Useful Resources
+
 * https://www.ruby-lang.org/ - The homepage of Ruby which has more links and some great tutorials.
 * https://rubyonrails.org/ - The homepage of Rails, also has links and tutorials.
 
-## Assigning Issues
+### Finding Issues to Work On
 
-We don't assign issues to individual contributors. You are welcome to work on any
-issue, and there's no need to ask first.
+> [!NOTE]
+> We don't assign issues to individual contributors. You are welcome to work on any issue, and there's no need to ask first.
 
-For more details see [our FAQ](FAQ.md)]
+For more details see [our FAQ](FAQ.md)
 
-## Coding style
+## How to Contribute
 
-We use [Rubocop](https://github.com/rubocop-hq/rubocop) (for ruby files)
-and [ERB Lint](https://github.com/Shopify/erb-lint) (for erb templates)
-to help maintain consistency in our code. You can run these utilities during
-development to check that your code matches our guidelines:
+Here's the typical contribution workflow:
 
-```
+1. **Find an Issue**: Browse our [issues](https://github.com/openstreetmap/openstreetmap-website/issues) or identify a bug/feature you'd like to work on
+2. **Fork & Clone**: Fork the repository and clone it to your local machine
+3. **Set Up**: Follow the [installation guide](INSTALL.md) to set up your development environment
+4. **Develop**: Make your changes following our [code quality guidelines](#code-quality-guidelines)
+5. **Test**: Write tests for your changes and ensure all existing tests pass
+6. **Commit**: Write clear commit messages following our [guidelines](#committing)
+7. **Submit a Pull Request**: Create a pull request with a clear description of your changes
+
+## Code Quality Guidelines
+
+### Coding Style
+
+We use [Rubocop](https://github.com/rubocop-hq/rubocop) (for ruby files) and [ERB Lint](https://github.com/Shopify/erb-lint) (for erb templates) to help maintain consistency in our code. You can run these utilities during development to check that your code matches our guidelines:
+
+```bash
 bundle exec rubocop
 bundle exec rails eslint
 bundle exec erb_lint .
 ```
```

Comment:
We should consider adding `bundle exec rails eslint:fix` to the list as well.
Eslint mentions a `--fix` option to fix any issues, but that doesn't work in our case (parameter is silently ignored).